src/ray/gcs/format/gcs.fbs

```diff
@@ -110,6 +120,12 @@ table ActorTableData {
   driver_id: string;
   // The ID of the node manager that created the actor.
   node_manager_id: string;
+  // Current state of this actor.
+  state: ActorState;
+  // Max number of times this actor should be reconstructed.
+  max_reconstructions: int;
+  // Remaining number of reconstructions.
+  remaining_reconstructions: int;
```

Comment:
A bit of a nit, but it occurs to me that for debugging purposes, it's probably more useful to know how many times the actor has been reconstructed so far than to know how many reconstructions are left. Since you can compute one from the other, maybe we should make this `num_reconstructions` instead? What do you think?